src/@chakra-ui/gatsby-plugin/semanticTokens.ts

```diff
@@ -101,6 +101,12 @@ const semanticTokens = {
         "linear-gradient(102.7deg, rgba(185, 185, 241, 0.2) 0%, rgba(84, 132, 234, 0.2) 51.56%, rgba(58, 142, 137, 0.2) 100%)",
     },
   },
+  shadows: {
+    buttonHover: {
+      _light: "4px 4px 0 var(--eth-colors-blue-100)",
+      _dark: "4px 4px 0 #352313",
+    },
+  },
```

Comment:
Nice, is the idea to delete the `shadows` file at some point? or should we still keep using that file to separate a bit the tokens.

Reply:
@pettinarip this is here because a semantic color is not being declared in the DS for this style of shadow, so a semantic has to be made for each color based on color mode.

Reply:
We need to use `primary.lowContrast` here but our `orange.800` is outdated.

We need to update `orange.800` to `#352313` and then we can use `primary.lowContrast`

Reply:
With the latest updates from the base branch, this has taken care of itself. Still have to push a commit to remove `buttonHover` from `semanticTokens`.